go.mod

```diff
@@ -8,9 +8,10 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/prometheus/client_golang v1.11.0
 	github.com/redhat-cne/rest-api v0.1.0
-	github.com/redhat-cne/sdk-go v0.1.0
+	github.com/redhat-cne/sdk-go v0.1.1-0.20210723134838-a808941830c9
```

Comment:
do we need to create a new release of the sdk-go for this or will this work? @aneeshkp

Reply:
no yet we will do after release or code freeze